ci/build_python.sh

```diff
@@ -18,9 +18,7 @@ rapids-logger "Building ucx-py"
 
 # Need `--experimental` flag to use `load_from_file` and `git.head_rev`
 rattler-build build --recipe conda/recipes/ucx-py \
-                    --experimental \
-                    --channel-priority disabled \
-                    --output-dir "$RAPIDS_CONDA_BLD_OUTPUT_DIR" \
+                    "${RATTLER_ARGS[@]}" \
```

Comment:
Might be useful to add in this PR how this `RATTLER_ARGS` comes about.

Reply:
Good call, done